### ISSUE TITLE
how to improve tts result 

### ISSUE THREAD

**Issue by freecui** [CorentinJ/Real-Time-Voice-Cloning#247]
![step-140000-align](https://user-images.githubusercontent.com/8288975/70890840-f59d7200-2020-11ea-86fe-4deb9ba85e50.png)
![step-140000-mel-spectrogram](https://user-images.githubusercontent.com/8288975/70890841-f6360880-2020-11ea-93ed-f86f05e67627.png)
this is my synthesizer result, is that ok ?
[result.zip](https://github.com/CorentinJ/Real-Time-Voice-Cloning/files/3966718/result.zip)
this is my vocoder result, and you can hear the result audio (130k_steps_5_gen_batched_target8000_overlap400) is not good, it seems noise, I don't know how to solve it?
I trained mandarin

**Comment by zhuxiaoxuhit** [comment]
Maybe need choose a overfit point with tocotron.

**Comment by zhuxiaoxuhit** [comment]
how many spks when training GE2E? If have good embedding, with 1000spks train tacotron will get good voice i think.

**Comment by freecui** [comment]
I didn't use so many speakers, and I think speaker numbers just affect the clone, not the synthesizer ; now my synthetic sound quality is bad, I think

**Comment by dazzlerrrr** [comment]
> I didn't use so many speakers, and I think speaker numbers just affect the clone, not the synthesizer ; now my synthetic sound quality is bad, I think

i also have this problem. but i think your quality is good, do you have trained three parts of this model by chinese datasets? and how many hours and how many speakers do your datasets have and what is the name of the Chinese database? and do you have midified some hyperparams, (such as: the num_mels? n_fft? hop_size? win_size? sample_rate?). and how about your clone results? hope for your  replying, thanks!!

**Comment by dazzlerrrr** [comment]
@freecui

**Comment by freecui** [comment]
I trained three parts of this modle by chinese datasets; I just use server hundreds speakers;

**Comment by dazzlerrrr** [comment]
did you change any hyper parameters? i also trained by chinese, but the synthesis model loss is very high. if is possible, can you share three models.

**Comment by dazzlerrrr** [comment]
@freecui, george_cql@163.com is my private email address, can you send me your wechat or communicate by email, that we can be more easier to communicate and optimize the model!

**Comment by zhuxiaoxuhit** [comment]
> ![step-140000-align](https://user-images.githubusercontent.com/8288975/70890840-f59d7200-2020-11ea-86fe-4deb9ba85e50.png)
> ![step-140000-mel-spectrogram](https://user-images.githubusercontent.com/8288975/70890841-f6360880-2020-11ea-93ed-f86f05e67627.png)
> this is my synthesizer result, is that ok ?
> [result.zip](https://github.com/CorentinJ/Real-Time-Voice-Cloning/files/3966718/result.zip)
> this is my vocoder result, and you can hear the result audio (130k_steps_5_gen_batched_target8000_overlap400) is not good, it seems noise, I don't know how to solve it?
> I trained mandarin

I donnot undersand y u got unbad synthetic voice with bad alignments.... Character embedding or one-hot u have used as input of taco?  
 if u want to know quality of synthesizer, maybe can make embedding by python script.  just a experiment for synthesizer.

**Comment by zhuxiaoxuhit** [comment]
Maybe  the text and the encoder embedding of this aynthetic audio  are in-set data?

**Comment by dazzlerrrr** [comment]
text and the encoder embedding are in-set processing in clone model, as the same as the tacotron model

**Comment by freecui** [comment]
130k is too short for wavernn training

**Comment by dazzlerrrr** [comment]
do you have tried changed the num_mels parameter, it is also important for mel spectrogram quality. when you tried the griffin-lim as the vocoder, less num_mels always generated poor quality audios.

**Comment by freecui** [comment]
yes, the traing result original mel spectrogram parameters are better

**Comment by blue-fish** [comment]
Closing this issue due to inactivity. Please reopen the issue if you have more to discuss.
